test/images/serve_hostname/VERSION

```diff
@@ -0,0 +1 @@
+v1.6
```

Comment:
I checked the gcr and see already multiarch image with different kind of versioning and see v1.6 as the highest version. To avoid any of these confusion let me mention v1.7